data/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can simplify to `<manifest package="com.fatalzero.rsandroid_task.data"/>`

Reply:
done